src/main/java/preponderous/ponder/minecraft/bukkit/nms/NMSAssistant.java

```diff
@@ -26,6 +24,15 @@ public NMSVersion getNMSVersion() {
      * @return {@code true} if it is.
      */
     public boolean isVersionGreaterThan(int version) {
+        return getNMSVersion().getMinor() > version;
+    }
+
+    /**
+     * Method to determine if the server version is greater than or equal to the given version.
+     * @param version to test against.
+     * @return {@code true} if it is.
+     */
+    public boolean isVersionGreaterThanOrEqualTo(int version) {
```

Comment:
that's prefect, thank you so much

Reply:
That's awkward I didn't see your response here lol